src/app.js

```diff
@@ -22,8 +22,9 @@ const localisePrice = (price, locale) => {
    * Implement me!
    * Hint: .toFixed() might help...
    */
+  price = price.toFixed(2).toString();
```

Comment:
This works, but I would generally advise assigning this to a new variable. Reassigning function arguments can be problematic. It changes a function's `arguments` property. Here's an article that explains this concept: https://spin.atomicobject.com/2011/04/10/javascript-don-t-reassign-your-function-arguments/.

Reply:
Got it! Will never do it again😄 Thanks!